.github/actions/is-release-in-progress-v1/src/is-release-in-progress.test.ts

```diff
@@ -0,0 +1,41 @@
+import { assert } from 'chai'
+import type { PullRequest } from './types'
+import isReleaseInProgress from './is-release-in-progress'
+
+export const BUG_PULL_REQUEST: PullRequest = {
+  id: 1,
+  state: 'open',
+  labels: [{ id: 12345, name: 'bug' }]
+}
+
+export const RELEASE_PULL_REQUEST: PullRequest = {
+  id: 2,
+  state: 'open',
+  labels: [{ id: 23456, name: 'release' }]
```

Comment:
Should add a test or two to ensure the exact match of `release`. So something like `release-2023` or `prerelease` doesn't return true